### ISSUE TITLE
Unused parameters in diag2index and anti2index

### ISSUE THREAD

**Issue by ilaripih** [basil00/Fathom#16]
Hi,

Is there a reason why these two functions (defined here: https://github.com/basil00/Fathom/blob/master/src/tbprobe.c#L293) take two parameters but `diag2index` does nothing with `d`, and as a result `anti2index` does nothing with `a`? Actually, why does `anti2index` even exist if it's just a wrapper for `diag2index` with the same signature?

**Comment by basil00** [comment]
Yes, I was wondering the same thing myself recently.  The move generator code was written a long time ago, even pre-dating the Fathom project, so I am not sure.  It is possible that they are just leftovers from earlier versions of the function.
